tst/EnergyPlus/unit/PlantLoopHeatPumpEIR.unit.cc

```diff
@@ -3499,6 +3499,7 @@ TEST_F(EnergyPlusFixture, Test_Curve_Negative_Energy)
     thisCoolingPLHP->loadSideInletTemp = 20;
     thisCoolingPLHP->sourceSideInletTemp = 20;
     thisCoolingPLHP->doPhysics(*state, curLoad);
+    thisCoolingPLHP->report(*state);
```

Comment:
called the new `report()` function here to update the report variables for testing.